fixtures/ios_app_with_custom_configuration/App/Sources/AppDelegate.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+import FrameworkA
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil
+    ) -> Bool {
+        window = UIWindow(frame: UIScreen.main.bounds)
+        let viewController = UIViewController()
+        viewController.view.backgroundColor = .white
+        window?.rootViewController = viewController
+        window?.makeKeyAndVisible()
+        
+        FrameworkA.hello()
+        
+        #if DEBUG_MACRO
```

Comment:
why not just use `#if DEBUG` rather than defining a custom macro?

Reply:
The reason behind choosing a custom debug compiler macro `DEBUG_MACRO` over the pre-defined `DEBUG` macro was that previously, `Debug` configuration was used by default, so a `DEBUG` macro was available. Having unique, custom compiler macro in custom configuration proofs that the changes work correctly, no related to default settings. Does that make sense? 😊

Reply:
yes! thanks